tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example/plugin.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A sample plugin to demonstrate reading scalars."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import csv
+import json
+import os
+import functools
+import mimetypes
+
+import six
+from werkzeug import wrappers
+import werkzeug
+
+from tensorboard import errors
+from tensorboard import plugin_util
+from tensorboard.backend import http_util
+from tensorboard.data import provider
+from tensorboard.plugins import base_plugin
+from tensorboard_plugin_example import metadata
+from tensorboard.util import tensor_util
+
+scalar_plugin_name = 'scalars'
+plugin_folder = '/data/plugin/example_raw_scalars/'
+
+class ExampleRawScalars(base_plugin.TBPlugin):
+  """Raw summary example plugin for TensorBoard."""
+
+  plugin_name = metadata.PLUGIN_NAME
+
+  def __init__(self, context):
+    """Instantiates ExampleRawScalars.
+
+    Args:
+      context: A base_plugin.TBContext instance.
+    """
+    self._multiplexer = context.multiplexer
+
+  def get_plugin_apps(self):
+    return {
+        '/index.js': self._serve_index,
+        '/scalars': self.scalars_route,
+        '/runInfo': self._serve_run_info,
+        '/static/*': self._serve_file,
+    }
+
+  @wrappers.Request.application
+  def _serve_run_info(self, request):
+    run_tag_mapping = self._multiplexer.PluginRunToTagToContent(
```

Comment:
Why is RunTagMapping important for TensorBoard? If I were reading this as a contributor, I think I would get a bit confused by why this data structure has to be created. Consider documenting its importance.